types/office-runtime/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
Please only disable the lint rules that you have to. It looks like you already have per-line lint rule excludes in the definition file -- prefer those over bulk disables whenever practical. Thanks!

Reply:
Yea, please don't copy/paste from other packages, see the `readme.md` for more info.